### DISCUSSION TITLE
Blurry text on latest version

### DISCUSSION THREAD

**Discussion by BatToko** [Zidras/DBM-Warmane#160] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/118014539/201403715-cf5d6c95-d6b8-4f9e-bbb9-711abbd6e53a.png)

With no changes to addons or the game whatsoever I get the blurry text on all notifications from DBM. Didn't have the issue on ver.7.07. Could be something not from the addon but from my game/other addons, that's why I put it in discussion and not issues.

**Comment by Zidras** [comment, marked as answer]
This is already covered in the FAQ section on my discord:
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/10605951/201488670-b3cd6bf9-639a-4c35-83ee-9ae68d733e48.png">

I suggest you start there first if you have any questions: https://discord.gg/CyVWDWS
